### ISSUE TITLE
v0.37 release plan

### ISSUE THREAD

**Issue by thanethomson** [cometbft/cometbft#41]
Was tendermint/tendermint#9091

This issue currently targets the v0.37 release.

In order to ship tendermint/tendermint#9053, we need to do the following:

- [x] tendermint/tendermint#9096 
- [x] Update documentation ~, including the roadmap~ (the roadmap will be added in a separate PR)
- [x] Define a release QA plan
   - [x] Define an integration testing plan with the Cosmos SDK (cc @marbar3778)
- [x] tendermint/tendermint#9053 
- [x] Additionally, we should aim to include as many of the addressed Q3 priorities in this release as possible:
   - [x] tendermint/tendermint#9077 
   - [x] tendermint/tendermint#9054
   - [x] tendermint/tendermint#9078 
- [x] First part of #409 (deprecate config to disable blocksync)
- [x] Identify and forward-port any necessary bug/security fixes from v0.35/v0.36 to v0.37
   - [x] tendermint/tendermint#9176 
   - [x] tendermint/tendermint#5447
   - [x] tendermint/tendermint#6755
   - [x] tendermint/tendermint#5728
   - [x] tendermint/tendermint#9287 
   - [x] tendermint/tendermint#6403
- [x] Cut `v0.37.x` branch
- [x] Cut release candidate
- [x] tendermint/tendermint#9429
- [x] Execute on the release QA plan and capture the results
   - [x] tendermint/tendermint#9123 
   - [x] tendermint/tendermint#9333
   - [x] #376
   - [x] #395
   - [x] Validate that there's no meaningful, substantial regression from v0.34 to v0.37
   - [x] Execute on the QA plan for integration with the Cosmos SDK
- [x] Cut final v0.37.0 release

Things that still need to be done before we "feature-freeze" v0.37 and cut the `v0.37.x` branch:
- [x] tendermint/tendermint#9300
- [x] tendermint/tendermint#9269 
- [x] tendermint/tendermint#9286 
- [x] Temporarily revert tendermint/tendermint#9175
- [x] tendermint/tendermint#9300
  - [x] tendermint/tendermint#9248 
  - [x] tendermint/tendermint#9244 
  - [x] tendermint/tendermint#9243 
- [x] tendermint/tendermint#9304
- [x] tendermint/tendermint#6403
